jwst/resample/tests/test_resample_step.py

```diff
@@ -3,14 +3,16 @@
 import numpy as np
 import pytest
 
-from jwst.datamodels import ImageModel
+from jwst.datamodels import ImageModel, ModelContainer
 from jwst.assign_wcs import AssignWcsStep
 from jwst.extract_2d import Extract2dStep
 from jwst.resample import ResampleSpecStep, ResampleStep
 
 @pytest.fixture
 def nirspec_rate():
-    shape = (2048, 2048)
+    ysize = 2048
```

Comment:
Hey, I thought we were supposed to keep test images *small*?!

Reply:
I think this needs to stay large because it is NIRSpec for some reason?  I didn't change it from the original test that was put here (not by me).  It still runs fast, because it is not doing any jump detection or ramp fitting.